share/library.mk

```diff
@@ -6,20 +6,22 @@ SHELL   := /usr/bin/env bash
 NAME    ?= $(notdir $(CURDIR:a/%=%))
 VERSION ?= $(shell sed -n 's/^version: *\(.*\)$$/\1/p' $(NAME).cabal)
 
-CABAL_SANDBOX_CONFIG := $(TOP)/cabal.sandbox.config
+CABAL_INSTALL_DEFARGS ?= -j --disable-documentation --disable-library-coverage
```

Comment:
Since this is a development Makefile I think we can safely assume the latest `cabal-install` is used, and set `--disable-coverage` as the default to avoid the obsolete flag.